src/main/resources/org/jenkinsci/plugins/workflow/support/steps/stash/StashStep/help.html

```diff
@@ -1,6 +1,7 @@
 <div>
     Saves a set of files for use later in the same build, generally on another node/workspace.
     Stashed files are not otherwise available and are generally discarded at the end of the build.
```

Comment:
This is a test comment.

Reply:
"preserveStashes" is a Declarative thing, it doesn't exist as part of this plugin: https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/7502d14aa4db891302abf48a2c7de7c9ced97f95/pipeline-model-definition/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/properties/PreserveStashesJobProperty.java#L53.

Reply:
I'm ok with @bitwiseman 's suggestion, and I'd commit it, but I'm not sure how much weigh does @dwnusbaum counter argument has. I'll wait for you guys to settle this.

Reply:
@dwnusbaum 
I see your point,  but I felt we needed to mention it, otherwise someone using this step as part of Declarative Pipeline might either assume they had to restash files or might not know how to preserve stashes are across run restarts.   How about this? 

```suggestion
    By default, stashed files are discarded at the end of a pipeline run.
    Other plugins may change this behavior to preserve stashes for longer.
    For example, Declarative Pipeline includes a <code>preserveStashes()</code> option
    to allow stashes from a run to be retain and used if that run is restarted. 
    <br>
```

Reply:
Yeah, this is the classic Jenkins problem of plugins not mapping well to holistic user-facing features. As long as you specify clearly that the option is part of Declarative, like in your latest suggestion, it seems fine to mention it.

Reply:
@ovidiub13
Okay, it sounds like that second suggestion looks good to dwnusbaum.

Reply:
Awesome. Please correct `from a run to be retain and` into `from a run to be retained and` and I'll commit it. I can't seem to change it, and I want to give you credit for that change and keep it your commit.

Reply:
@ovidiub13 Done.